dotCMS/src/main/java/com/dotcms/browser/BrowserAPIImpl.java

```diff
@@ -60,6 +62,7 @@
     @CloseDBIfOpened
     public Map<String, Object> getFolderContent(final BrowserQuery browserQuery) throws DotSecurityException, DotDataException {
 
+        final MutableInt countItems                     = new MutableInt(0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5476854)